src/benchmarks/beluga_vs_nav2_multi_dataset/reports/nominal_report/sections/datasets/magazino_datasets_description.inc

```diff
@@ -0,0 +1,34 @@
+Dataset information
+^^^^^^^^^^^^^^^^^^^
+
+These results are based on a modified version of the Cartographer Magazino dataset,
+which was repurposed to serve as a 2D localization test scenario. The original
+dataset is available `here <https://github.com/magazino/cartographer_magazino>`_.
+
+There's little information about this dataset. In the repository linked
+above it is mentioned that the dataset was recorded
+in a hallway using a Magazino `TORU robot <https://robotsguide.com/robots/toru>`_.
+
+The set contains only two bagfiles, corresponding to two different trajectories along the same hallway.
+"Hallway Localization" is a bit over two minutes long, while "Hallway Return" is a little below four
+minutes long.
```

Comment:
@glpuga nit: shall we use numbers to talk about duration for consistency? Or maybe use both letters and numbers?

Reply:
Numbers sounds good.